sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -164,7 +165,10 @@ public static void updateMetricPointBuilder(
 
         metricTelemetryBuilder.setMetricPoint(pointBuilder);
 
+        // remove any attributes that start with "applicationinsights.internal."
         Attributes attributes = pointData.getAttributes();
+        attributes.asMap().keySet().removeIf(it -> it.getKey().startsWith("applicationinsights.internal."));
```

Comment:
I think you were right about this already being excluded

Reply:
we were using EMPTY_MAPPING and didn't ignore the standard metrics attribute prefixes.

Reply:
do we want to ingore all these prefixes for metrics: 

 "http.",
                "db.",
                "message.",
                "messaging.",
                "rpc.",
                "enduser.",
                "net.",
                "peer.",
                "exception.",
                "thread.",
                "faas.",
                "code.",
                "job.", // proposed semantic convention which we use for job,
                "applicationinsights.internal."));